doc/sphinxman/source/psithonmol.rst

```diff
@@ -162,6 +162,7 @@ provided in the molecule block :samp:`molecule {optional_molecule_name} \\{...\\
    line of the molecule block, they are interpreted as the molecular charge
    and multiplicity (:math:`2 M_s + 1`), respectively. For multi-fragment 
    complexes, each fragment can have a :samp:`{charge} {multiplicity}` line.
+   If not explicitly provided, each fragment is assumed to be :samp:`{0} {1}`.
```

Comment:
Wait, you can specify a charge for the _entire system_ independent of any particular fragments?

Reply:
yes, also a post-2018 feature (and a q-chem feature).

```
0 1
--
+1 1
H
--
-1 1
H
```

Reply:
https://github.com/psi4/psi4/blob/master/tests/isapt2/input.dat#L6-L31

Reply:
So, we require that each fragments needs to have the high spin multiplicity given its charge, and that the entire system is in the high-spin combination of each fragment?

Reply:
Not really _require_ since it'll allow low-spin solutions if you fully (tot and frag) specify them.

I'm having a hard time with a concise description. Three descriptions, one for no chgmult given, one for total given, and one for frags-only given (possible wording below), might be easier. To me at least, reading https://github.com/MolSSI/QCElemental/blob/master/qcelemental/molparse/chgmult.py#L110-L136 makes most sense, but I daresay that's familiarity.

* frags-only given: The reasonability of each fragment's charge and multiplicity will be checked (paired-ness, enough electrons, ghost status, etc.). (This will require high-spin rules unless both total and fragments charge and mult fully given.) Then the total charge and multiplicity will be the sum and high-spin sum of the fragments, respectively.

Reply:
In Q-Chem one can also define negative multiplicities, which may be useful for e.g. constructing antiferromagnetic couplings.

Reply:
Please keep desired Psi features separate from this discussion about _describing the features Psi currently has_.

Reply:
> In Q-Chem one can also define negative multiplicities, which may be useful for e.g. constructing antiferromagnetic couplings.

https://manual.q-chem.com/5.2/Ch13.S2.html , eek! I see the utility, and it would be grouping fragment info where the frags specified. Logically it wouldn't be too hard, I don't think. But handling the opt-in of those solution pathways and then reconciling the output with any other alpha/beta specification, say docc/socc, sounds tricky. definitely glad to know that specification convention exists, though.

Reply:
Let's say that I specify
```
1 2
--
H
--
Li
```

How do I determine whether the cation is H or Li? What if I swapped the order of Li and H?

Reply:
Yes, it'll solve differently if you swapped the order. Cation lands on first fragment. https://github.com/MolSSI/QCElemental/blob/master/qcelemental/molparse/chgmult.py#L135 . If the charge had been -2, it'd land on Li regardless of order.